src/planwise/client/projects/components/sidebar.cljs

```diff
@@ -95,7 +124,9 @@
          :label-fn :name
          :on-change #(dispatch [:projects/set-transport-time %])
          :model transport-time]
-        (icon :car)]])))
+        (icon :car)]
+       (when @wizard-mode-on
+         (next-and-back-buttons :transport (:id @current-project)))])))
```

Comment:
I'd try to move all three invocations of ` (when @wizard-mode-on (next-and-back-buttons tab-name (:id @current-project)))` from each tab-filters method to `sidebar-section`, something more or less like:

```
(defn sidebar-section [selected-tab]
  [:aside
   (case selected-tab
     :demographics [demographics-filters]
     :facilities   [facility-filters]
     :transport    [transport-filters])])
   (when @wizard-mode-on
     (next-and-back-buttons selected-tab (:id current-project)))])))
```